### ISSUE TITLE
PatternFly homepage revamp

### ISSUE THREAD

**Issue by evwilkin** [patternfly/patternfly-org#3373]
Follow up to design issue https://github.com/patternfly/patternfly-design/issues/1229, details to come

**Comment by lboehling** [comment]
Here is a link to the latest design mocks: https://www.sketch.com/s/fd4e10c4-195f-4d9e-8ec7-dd90d9e2ca12. We've decided to go ahead and move forward with the homepage ahead of the Penta theme availability, so I've updated the mocks to reflect our classic theme. Final copy will be documented [here](https://docs.google.com/document/d/1cFNqaVCfLA6aArJgZOqsfBFiftvTnyBLQ_io8QB25yk/edit?usp=sharing). 

I still need to finalize the assets for the component showcase, which will be handed off as a single image. Will add the svgs for those to this issue once completed!

**Comment by mcarrano** [comment]
One of the requirements for the new homepage is to include a live feed from the PatternFly Medium blog so the 4 most recent posts will always be exposed. I did a little bit of research on this and seems like it should be possible. I found [this article](https://www.twilik.com/embed-your-medium-blog) that includes a demo and a link to an open source GitHub here: https://github.com/chrisj74/vue-rss-blog It would be great to do a PoC to see if this will work with the PatternFly site.
